src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -16,32 +16,114 @@ tags:
+
+[PageSpeed Insights](https://pagespeed.web.dev/) will show both the user-perceived CLS from a URL where it exists in the "Discover what your real users are experiencing" section, and also the lab-based load CLS in the "Diagnose performance issues" section beneath. If you see a difference between these, this is likely caused by post-load CLS.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/2eO9BsIb8Gx6kU4Zvbqa.png", alt="Screenshot of PageSpeed Insights showing URL-level data highlighting the real user CLS which is considerably larger than the Lighthouse CLS", width="800", height="450" %}
+
+{% Aside important %}
+  PageSpeed Inights will show URL-level data where it exists, and attempt to fall back to origin-level data where this does not exist. Always check what data is showing to ensure you do not waste time trying to track down a CLS issue that actually exists on other pages on your origin! In the above example you can see this is URL-level data as shown in the top right of the image.
+{% endAside %}
+
+### Identifying load CLS issues
+
+When the CrUX and Lighthouse CLS scores of PageSpeed Insights are broadly in line, this usually indicates there is a load CLS issue that was detected by Lighthouse. In this case Lighthouse will help with two audits to provide more information on images causing CLS due to missing width and height, and also list all the elements that shifted for the page load along with their CLS contribution.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/2C3v6dGwPx2yFyYpVdue.png", alt="Lighthouse Screenshot showing the CLS audits providing more information to help you identify and address CLS issues", width="800", height="544" %}
+
+{% Aside important %}
+  Lighthouse will identify the elements that were shifted, but often these are the ones _impacted_ rather than the elements _causing_ the CLS. For example, if a new element is inserted into the DOM, the elements that are beneath it will show in this audit, but the root cause is the addition of the new element above. Hopefully, the shifted element will be sufficient to help you identify and resolve the root cause.
+{% endAside %}
+
+### Identifying post-load CLS issues
+
+When the CrUX and Lighthouse CLS scores of PageSpeed Insights are not in line, then this likely indicates post-load CLS. To replicate that, the [Web Vitals Chrome extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma) can be used to monitor CLS as you interact with a page, either in a heads up display, or in the console.
+
+Alternatively, you can browse your web page while [recording CLS using a JavaScript snippet](/cls/#measure-cls-in-javascript) pasted into the console.
```

Comment:
WDYT about linking "monitor CLS" in the previous paragraph to the snippet source? That addresses my concerns while I think addressing yours about supporting advanced use cases.

Reply:
Sorry I don’t understand what you  mean by linking to the snippet source? Do you mean link to the web-vitals extension source instead of the CLS docs?

Reply:
Suggested the change

Reply:
I think that's more confusing. I think we've made it clear here the preferred way is the extension but think we should offer the alternative for those that cannot use it ([eg1](https://twitter.com/DanShappir/status/1642430405813870592?s=20), [eg2](https://webperformance.slack.com/archives/C04BK7K1X/p1678197673330799?thread_ts=1678197553.113529&cid=C04BK7K1X)), or do not want to use the extension.

Reply:
In #9845 we've removed the full, complex, CLS snippet and refer readers to the web-vitals source code instead for that. We have, however, added a simpler, layout shifts snippet to that code and this PR refers to that.